hi-ml-histopathology/src/histopathology/preprocessing/create_panda_tiles_dataset.py

```diff
@@ -217,14 +246,69 @@ def main(panda_dir: Union[str, Path], root_output_dir: Union[str, Path], level:
 
     logging.info("Merging slide files in a single file")
     merge_dataset_csv_files(output_dir)
+    logfile.close()
 
 
 if __name__ == '__main__':
-    main(panda_dir="/tmp/datasets/PANDA",
-         root_output_dir="/datadrive",
-         level=1,
-         tile_size=224,
-         margin=64,
-         occupancy_threshold=0.05,
-         parallel=True,
-         overwrite=False)
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--container-dir",
+        type=str,
+        required=True,
+        help="Folder with the 'datasets' container",
+    )
+    parser.add_argument(
+        "--output-dir",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "--level",
+        type=int,
+        default=1,
+    )
+    parser.add_argument(
+        "--size",
+        type=int,
+        default=224,
+    )
+    parser.add_argument(
+        "--margin",
+        type=int,
+        default=64,
+    )
+    parser.add_argument(
+        "--occupancy",
+        type=float,
+        default=0.05,
+    )
+    parser.add_argument(
+        "--no-parallel",
+        action="store_false",
+        default=True,
+        dest="parallel",
+    )
+    parser.add_argument(
+        "--overwrite",
+        action="store_true",
+        default=False,
+    )
+    parser.add_argument(
+        "--filter-slide",  # process only this slide
+        type=str,
+        default="",
+    )
```

Comment:
This was useful for debugging; I can remove it if necessary.

Reply:
Should be fine to leave it in, IMO. I also had a similar param when developing this script.